packages/nimble-components/src/table-column/number-text/tests/table-column-number-text.spec.ts

```diff
@@ -267,4 +275,101 @@ describe('TableColumnNumberText', () => {
             expect(pageObject.getGroupHeaderTitle(0)).toBe('');
         });
     });
+
+    describe('alignment', () => {
```

Comment:
nit: Test output may read better without this describe block. I don't think it's functionally necessary.